tests/lean/run/optionGetD.lean

```diff
@@ -0,0 +1,12 @@
+import Lean.Data.HashMap
+
+def test (m : Lean.HashMap Nat Nat) : IO (Nat × Nat) := do
+  let start := 1
+  let mut i := start
+  let mut count := 0
+  while i != 0 do
+    i := (m.find? i).getD (panic! "key is not in the map")
```

Comment:
Would this test actually fail on panic? It's not an output test and we're not running them under `LEAN_ABORT_ON_PANIC`

Reply:
Really? I would have expected panics to be treated as failures in `run/` tests

Reply:
I moved it to `tests/lean` and confirmed that the output contains `PANIC` on master.

Reply:
> I would have expected panics to be treated as failures in `run/` tests

Perhaps they should be but right now we only set the flag for the build itself